app/controllers/manage_user_controller.rb

```diff
@@ -5,4 +5,10 @@ def index
     @users = User.all
     @articles = Article.all
   end
+
+  def user_roles
```

Comment:
change the method name, You are fetching user

Reply:
I removed this method (unwanted method)